.github/workflows/pr.yaml

```diff
@@ -11,6 +11,12 @@ jobs:
         include:
           - isUpgrade: true
             ref: main
+        k8sVersion:
+          - v1.21.1
+          - v1.20.7
+          - v1.19.11
+          - v1.18.19
+          - v1.17.17
```

Comment:
would be neat to specify this in a way that @dependabot understands

Reply:
It wouldn't be Dependabot, but if there's a published json file of versions somewhere we could feed that in.

Reply:
This is close https://github.com/kubernetes/website/blob/master/data/releases/schedule.yaml -- not sure how to convert that to kind versions.